command_server/generic/routes.py

```diff
@@ -0,0 +1,48 @@
+from aiohttp.web_urldispatcher import UrlDispatcher
+
+from generic.base import BaseView
+
+ROUTES = []
+
+
+def dispatch(view, method):
+    def _dispatch(request, *args, **kwargs):
+        kwargs.update(request.match_info)
+        return view.dispatch(request, method, *args, **kwargs)
+    return _dispatch
+
+
+def route(path_reg, method='*'):
+    """
+    Route decorator for simple handlers
+
+    :param path_reg: string
+    :param method: string
+    :return:
+    """
+    def _dec(func): # maybe it should be as coroutine?
+        ROUTES.append((method, path_reg, func))
+
+        def _run(*args, **kwargs):
+            return func(*args, **kwargs)
+        return _run
+    return _dec
+
+
+def class_route(path_reg):
+    """
+    Route decorator for class-based views
+
+    :param path_reg: string
+    :return:
+    """
+    def handler_builder(cls):
+        assert issubclass(cls, BaseView), 'Class {!r} should inherit BaseView'.format(cls)
+        view = cls.as_view()
+        for method in UrlDispatcher.METHODS:
+            method_l = method.lower()
+            method_handler = getattr(view, method_l, None)
+            if callable(method_handler):
+                setattr(view, method_l, route(path_reg, method)(dispatch(view, method_handler)))
```

Comment:
Strongly Yes! It is good solution. Actually there no a lot decorators applied. Why you scared decorators, they are quite sweet:) Check out django's code, they use much more decorators that you can imagine